foundation/foundation-mda/src/main/java/com/boozallen/aiops/mda/metamodel/element/python/PythonStep.java

```diff
@@ -135,8 +136,9 @@ private Set<String> getImportsWithRelativePaths(boolean isImplModule) {
         for (String moduleImport : imports) {
             if (moduleImport.startsWith("from record.")
                     || moduleImport.startsWith("from dictionary.")) {
+                String rootArtifact = PythonGeneratorUtils.rootArtifactId.replace("-", "_");
                 importsWithRelativePaths.add(moduleImport.replace("from ",
-                        isImplModule ? "from .." : "from ..."));
+                        "from " + rootArtifact + "_data_records."));
```

Comment:
I: few of python imports had relative imports, because they did generated data records under pyspark directory.

Now that we deleted duplicated data record in pyspark, needed to point those into correct package.